000-vip-init.php

```diff
@@ -248,6 +248,14 @@
 	$headers['X-Powered-By'] = 'WordPress VIP <https://wpvip.com>';
 	$headers['Host-Header']  = 'a9130478a60e5f9135f765b23f26593b'; // md5 -s wpvip
 
+	// By default, the 'vip_index_noprod_domains' filter returns an empty array (to prevent indexing any non-production sites).
+	$index_noprod_domains = apply_filters( 'vip_index_noprod_domains', '__return_empty_array' );
```

Comment:
The second argument for `apply_filters` would be the value, so in the code right now it'd be this:

```
wp> apply_filters( 'foo', '__return_empty_array' );
=> phar:///usr/local/bin/wp/vendor/wp-cli/shell-command/src/WP_CLI/Shell/REPL.php:52:
string(20) "__return_empty_array"
```

```suggestion
	$index_noprod_domains = apply_filters( 'vip_index_noprod_domains', [] );
```